src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.Divide.cs

```diff
@@ -70,11 +72,80 @@ public static void Divide<T>(T x, ReadOnlySpan<T> y, Span<T> destination)
         internal readonly struct DivideOperator<T> : IBinaryOperator<T> where T : IDivisionOperators<T, T, T>
         {
             public static bool Vectorizable => typeof(T) == typeof(float)
-                                            || typeof(T) == typeof(double);
+                                            || typeof(T) == typeof(double)
+#if NET10_0_OR_GREATER
+                                            || ((Avx512F.IsSupported || Avx.IsSupported) && typeof(T) == typeof(int));
```

Comment:
Does Avx.IsSupported not imply Avx512F.IsSupported?

Reply:
Avx.IsSupported does not imply Avx512F.IsSupported. There is hardware that supports Avx/Avx2, but not Avx512.

The Vector128 path only needs Avx, it does not need Avx512. A separate check for Avx allows for some work to get vectorized on machines that don't have Avx512.

Reply:
Sorry, what I meant was, why is the first condition here necessary? If Avx512F.IsSupported is true, is it not guaranteed that Avx.IssSupported is also true?

Reply:
Oh, I see. Yes, you're right. I think this can be simplified by removing Avx512F.IsSupported since the minimum requirement for vectorizing the operation is just Avx.IsSupported.

I'll make the adjustment.